data/saas/config/mandrill_config.yml

```diff
@@ -0,0 +1,44 @@
+saas_config:
+  fides_key: <instance_fides_key>
+  name: Mandril SaaS Config
+  type: mandrill
```

Comment:
One thing right up front, which is annoying but we should do now: let's start calling this Mailchimp 😄 

Mandrill doesn't actually exist any more, all the API docs are "Mailchimp Transactional Email" like this: https://mailchimp.com/developer/transactional/api/

Reply:
That's perfect, fewer files to create, we can just add the consent section to existing mailchimp config